crates/bevy_tasks/src/slice.rs

```diff
@@ -15,6 +42,35 @@ pub trait ParallelSlice<T: Sync>: AsRef<[T]> {
         })
     }
 
+    /// Splits the slice into a maximum of `self.len() / max_tasks` chunks of approximately
```

Comment:
This sentence runs on a bit for my tastes

Reply:
Tried to reword it more succinctly. Hope this is better.